change/@fluentui-react-components-6ac4a4fd-0081-4093-b47e-17fab5cfae9c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "'feat:",
```

Comment:
this is wrong, pleas fix it

Reply:
fixed

Reply:
I pushed it together with a wrong dependency fix